glue/viewers/image/qt/data_viewer.py

```diff
@@ -38,7 +39,9 @@ class ImageViewer(MatplotlibImageMixin, MatplotlibDataViewer):
 
     tools = ['select:rectangle', 'select:xrange',
              'select:yrange', 'select:circle',
-             'select:polygon', 'image:point_selection', 'image:contrast_bias',
+             'select:polygon', 'image:point_selection',
+             'image:pixel_extraction',
```

Comment:
If we want to add this by default, we need to make sure the icon is different from point_selection (at the moment they might be the same?)

Reply:
The icon for the pixel extraction tool looks similar but not the same as the point selection tool, it was designed like so on purpose to reflect their similar functionality. 

<img width="26" alt="Screenshot 2020-08-18 at 3 32 17 AM" src="https://user-images.githubusercontent.com/26264600/90436594-88883200-e103-11ea-9eb9-163c5ecc4c27.png">

Reply:
> The icon added looks identical to https://github.com/glue-viz/glue/blob/master/glue/icons/glue_crosshair.png?

I think they are not really identical, except maybe there are two different pixel extraction tool icons right now with different icons, one from `glue` and the other from `glue-solar` (with the latter to be removed once the former can be upstreamed), so I think you might have been referring to this second icon.